portfolio/src/main/webapp/homePage.css

```
@@ -0,0 +1,143 @@
+html{
+    scroll-behavior: smooth
+}
+
+body{
```

Add a space between the css selector and the opening `{`. In the rules below as well.
```suggestion
html {
    scroll-behavior: smooth
}

body {
```